probablefutures-py/probablefutures/probablefutures.py

```diff
@@ -0,0 +1,48 @@
+import requests
+import input
```

Comment:
After building the package and trying to use it, it didn't find the input module. I had to change it to:
`import probablefutures.input as input`

Reply:
You are probably importing at the wrong level.  The library is mean to be imported with "probablefuture-py" as the root, so right now, you have to do:  "from probablefutures.probablefutures import ProbableFutures" which is awkward.  I'll work on make all this nicer.